### PR TITLE
fix: add password requirements

### PR DESCRIPTION
**Pull request by schalkneethling**
Add password requirements to create account form. The user of `aria-describedby` here is to ensure that a screen reader will read the password requirements when a user focuses on the password field.

### CODE REVIEW

**Comment by schalkneethling** [comment]
I kept the parcel-bundler upgrade in a separate commit as I am not sure exactly how it happened :) What seems to have happened is that the first time I ran `yarn dev`, it first upgraded `parcel-bundler` and then started the dev server etc.

Perhaps this is a new "feature" of Parcel?

**Comment by schalkneethling** [comment]
> I kept the parcel-bundler upgrade in a separate commit as I am not sure exactly how it happened :) What seems to have happened is that the first time I ran `yarn dev`, it first upgraded `parcel-bundler` and then started the dev server etc.
> 
> Perhaps this is a new "feature" of Parcel?

Nope, that was not it. I am baffled as to why/how this happened but, perhaps it is ok?

File: src/js/components/account/user-registration.js [hunk 1]
```diff
@@ -22,10 +23,39 @@ function formatError(jqXHR) {
...
+  });
+
+  heading.textContent = "Your password must:";
+  container.appendChild(heading);
+  container.appendChild(uList);
+
+  container.setAttribute("id", "password-requirements");
+
+  return container;
+}
```
Comment (jbothma) [inline src/js/components/account/user-registration.js:49]: would $("<div><h5>...") not have been much more readable? :)

**Review by jbothma — CHANGES_REQUESTED** [review]
it looks like a parcel downgrade? could you just revert that change manually? Other than that this looks good - nice touch on the aria attribute - I'm learning lots

File: package.json [hunk 2]
```diff
@@ -18,7 +18,6 @@
     "http-server": "^0.12.3",
     "import-webflow-export": "^0.0.4",
     "npm-run-all": "^4.1.5",
-    "parcel": "^1.12.4",
```
Comment (schalkneethling) [inline package.json:21]: not sure how this ended up in here but, we do not need both right? Everything seems to work just fine with only `parcel-bundler`. This is also what the docs say https://parceljs.org/getting_started.html :)

Comment (jbothma) [reply package.json:21]: agreed

**Comment by jbothma** [comment]
Looks like it has a conflict - I can't resolve it quickly here, so I'll leave it inyour hands.

**Review by jbothma — APPROVED** [review]

**Pull request merged**